_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit-Machines 244.
```

Comment:
"Cockpit-Machines" looks kinda funny...  Is that actually how we write it?

Reply:
I can also change it to Cockpit Machines.

Reply:
We haven't standardized how we refer to it actually. Same for podman.

Reply:
Removed the hiphen.